private/pkg/app/appcmd/appcmd.go

```diff
@@ -261,6 +271,11 @@ func commandToCobra(
 		Hidden:     command.Hidden,
 		Short:      strings.TrimSpace(command.Short),
 	}
+	cobraCommand.SetHelpTemplate(`{{.Short}}
```

Comment:
Can you explain why this needs to change?

Reply:
It had to do with adding the `Usage` section in between the `short` and `long` description. 
Adding the Usage field after the short description means that the structure of the command is the thing that users see first instead of after the description where there might already be a many examples.

Without change:
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/32605850/218202861-ea4ff14d-b9d7-4acc-b6b7-271eef6ff444.png">

With change:
<img width="1157" alt="image" src="https://user-images.githubusercontent.com/32605850/218202994-ee06a340-0822-488c-a0ef-e8e9b94d8aff.png">

Reply:
Also the change that I made is identical in the help text